docs/source/tasks/table_top_gripper/index.md

```diff
@@ -282,3 +282,27 @@ One robot must push/give the cube on its side to the other side so the other rob
 <video preload="auto" controls="True" width="100%">
 <source src="https://github.com/haosulab/ManiSkill/raw/main/figures/environment_demos/TwoRobotStackCube-v1_rt.mp4" type="video/mp4">
 </video>
+
+## RollBall-v1
+![dense-reward][reward-badge]
+
+:::{dropdown} Task Card
+:icon: note
+:color: primary
+
+**Task Description:**
+A simple task where the objective is to push and roll a ball to a goal region at the other end of the table.
+
+**Supported Robots: (Panda, Panda)**
```

Comment:
this is not right, code says only the single agent "Panda" is supported. (Panda, Panda) indicates two pandas at once

Reply:
Fixed it